lib/web_ui/test/canvaskit/path_metrics_test.dart

```diff
@@ -0,0 +1,43 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// @dart = 2.6
+import 'package:test/test.dart';
+
+import 'package:ui/src/engine.dart';
+import 'package:ui/ui.dart' as ui;
+
+void main() {
+  setUpAll(() async {
+    await ui.webOnlyInitializePlatform();
+  });
+
+  test('Using CanvasKit', () {
+    expect(experimentalUseSkia, true);
+  });
+
+  test(SkPathMetrics, () {
```

Comment:
At the end I do `path.addOval` and check that we now have 2 metrics. Does it not cover this case?

Reply:
No, because that test would still have passed (ignoring the off-by-one error) before. What we want to show is that you have have 2 valid PathMetrics that you can query at the same time

Reply:
Gotcha. Done.